freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/verticalscale/event/FreeIPAVerticalScaleEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.freeipa.flow.freeipa.verticalscale.event;
+
+import com.sequenceiq.flow.core.FlowEvent;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+
+public enum FreeIPAVerticalScaleEvent implements FlowEvent {
+    STACK_VERTICALSCALE_EVENT("STACK_VERTICAL_SCALE_TRIGGER_EVENT"),
+    STACK_VERTICALSCALE_FINISHED_EVENT(EventSelectorUtil.selector(FreeIPAVerticalScaleResult.class)),
+    STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT(EventSelectorUtil.failureSelector(FreeIPAVerticalScaleRequest.class)),
```

Comment:
this is not in sync with the handler I think. Also I would expect this to be used in `STACK_VERTICALSCALE_FINISHED_STATE`, but it's used in `STACK_VERTICALSCALE_STATE`, but maybe I missed something